### PR TITLE
Host LightenDiffusion on Hugging Face

### PR DESCRIPTION
**Pull request by NielsRogge**
Hello @JianghaiSCU 🤗 

I'm Niels and work as part of the open-source team at Hugging Face. I discovered through ECCV (congrats!) and indexed the paper page here: https://huggingface.co/papers/2407.08939. The paper page lets people discuss about your paper and lets them find artifacts about it (your model for instance) you can also claim the paper as yours which will show up on your public profile at HF.

Would you like to host the model you've pre-trained on https://huggingface.co/models, rather than Google Drive? Hosting on Hugging Face will give you more visibility. We can add tags in the model cards so that people find the models easier, link it to the paper page, etc.

If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). If it's a PyTorch model, you can use [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds from_pretrained and push_to_hub to the model which lets you to upload the model and people to download and use models right away.
If you do not want this and directly want to upload model through UI or however you want, people can also use [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).

After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your model.

You can also build a demo to your model on [Spaces](https://huggingface.co/spaces) we can provide you an A100 grant.

What do you think?

### CODE REVIEW

**Comment by nanqiaobei** [comment]
(1) Regarding the issue with code execution and data loading: The first problem is the lack of a .txt file for input data. The second issue is that the input images have different sizes, and the code does not perform resizing to a uniform scale. How is the code running despite these issues?

(2) The training configuration for the first stage of the CTDN model has not been made publicly available. Could you please share the training configuration for this part?

(3) I directly ran the model using randomly generated data with the shape (4, 6, 244, 244).

(4) In the forward pass of the DiffusionUNet model, there is an error at torch.cat([h, hs.pop()], dim=1), temb. Could you provide guidance on resolving this issue?
